docker/docker-compose.override.yml

```diff
@@ -51,22 +51,6 @@ services:
     depends_on:
       - database
   ldap:
-    image: bitnami/openldap:2.6.9
-    environment:
-      - LDAP_ADMIN_PASSWORD=admin
-      - LDAP_PORT_NUMBER=389
-      - LDAP_SKIP_DEFAULT_TREE=yes
-    volumes:
-      - "./ldap_ldifs:/ldifs"
+    build: ../components/testldap
```

Comment:
Do we also need the testldap container in the helm chart for testing?

Reply:
We didn't use it before. We only build and start Quality-time, but don't run any tests in the helm-chart workflow. I've created an issue for this: #11887.